<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactsEditName</class>
 <widget class="QDialog" name="ContactsEditName">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit contact name</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/contacts</normaloff>:/contacts</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="title_label">
       <property name="text">
        <string notr="true">Title:</string>
       </property>
       <property name="buddy">
        <cstring>title</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="title"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="first_name_label">
       <property name="text">
        <string notr="true">First name:</string>
       </property>
       <property name="buddy">
        <cstring>first_name</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="first_name"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="last_name"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="suffix_label">
       <property name="text">
        <string notr="true">Suffix:</string>
       </property>
       <property name="buddy">
        <cstring>suffix</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="suffix"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="last_name_label">
       <property name="text">
        <string notr="true">Last name:</string>
       </property>
       <property name="buddy">
        <cstring>last_name</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ContactsEditName</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ContactsEditName</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
